lib/instrumentation/index.js

```diff
@@ -58,6 +58,20 @@ function Instrumentation (agent) {
       return this.bindingSpan || this.activeSpan
     }
   })
+
+  this._modules = []
+  this._patches = {}
+
+  for (let mod of MODULES) {
+    this.registerPatch(mod, () => require(`./modules/${mod}`))
+  }
+}
+
+Instrumentation.prototype.registerPatch = function (name, handler) {
+  this._modules.push(name)
+  this._patches[name] = typeof handler !== 'function'
```

Comment:
🤔 I think I would probably change this check to a `typeof handler === 'string'` - but it's just a gut feeling... maybe it doesn't matter

Reply:
Actually, probably better to explicitly check for both strings and functions, then also log an error if it is neither.

Reply:
I changed it to check both and log if it's neither.